doc/api/crypto.md

```diff
@@ -480,6 +480,36 @@ of the plaintext in bytes. See [CCM mode][].
 
 The `decipher.setAAD()` method must be called before [`decipher.update()`][].
 
+Example: Using decipher.setAAD() method:
```

Comment:
Since we're in the `decipher.setAAD()` section, pointing out that the code is example code for that method is unnecessary. Would you mind removing this text?

Reply:
I just removed the text, thanks for your feedback.